Documentation/ApiOverview/ContentElements/Index.rst

```diff
@@ -110,21 +110,86 @@ your projects.
 possible without writing PHP code and can therefore also be done by
 TYPO3 integrators.
 
+..  _plugins:
 
+Plugins in TYPO3
+----------------
+
+A **plugin** in TYPO3 is a more complex module, typically providing dynamic
+or interactive functionality. Plugins are usually provided by extensions
+that introduce new features to the website.
+
+The data to be displayed is usually supplied by a special PHP class
+called a "controller". Depending on the technology used in the controller
+the plugin can be an Extbase plugin or a plain plugin.
 
-..  _plugins:
 ..  _plugins-extbase:
-..  todo: Introduce Extbase plugins
 
-What are plugins?
------------------
+Extbase plugins
+~~~~~~~~~~~~~~~
+
+For usage in the TYPO3 backend Extbase plugins are registered with utility
+functions of class :php:`\TYPO3\CMS\Extbase\Utility\ExtensionUtility` (not to
+be confused with :php:`\TYPO3\CMS\Core\Utility\ExtensionManagementUtility`).
+
+An Extbase plugin is configured for the frontend with
+:php:`ExtensionUtility::configurePlugin()` in file
+:file:`EXT:my_extension/ext_localconf.php`:
+
+..  literalinclude:: _Plugins/_ext_localconf_extbase_plugin.php
+    :caption: EXT:my_extension/ext_localconf.php
+
+By using `ExtensionUtility::PLUGIN_TYPE_PLUGIN` as fifth parameter is is also
+possible to add the plugin as a list type. See :ref:`plugins-list_type`.
```

Comment:
This is the default

Reply:
Understand. You're using CONTENT_ELEMENT in the example and in futute CONTENT_ELEMENT will be the new default.

Reply:
at least so I hope. For the time being we want to suggest using CONTENT_ELEMENT without it being the default yet